### ISSUE TITLE
Bucket streams and the future of better messaging

### ISSUE THREAD

**Issue by mcbyrne** [kentcdodds/ama#53]
Hi,
I had a similar if not identical concept with what I call smash tags but have also extended it. 
Why are you dropping the idea? 
To what extent do you still have an interest in improving messaging?

**Comment by kentcdodds** [comment]
[Context](https://twitter.com/kentcdodds/status/673707858848972800). [Demo](https://www.youtube.com/watch?v=xvWZSNiYmcw)

Thanks for the question @mcbyrne! The project was for my capstone project for my masters degree at [BYU](http://marriottschool.byu.edu/mism/). I was totally planning on taking it all the way and I worked on it a lot. I haven't continued with the idea since then because I just don't have the time or willpower to try to make it a real thing. I still totally love the idea and I wish it existed. I just don't want to take the risk on it and quit my job to work on it full time.

So, am I still interested in the idea. Totally. Will I do anything about it. Probably not. But if you're interested in what I've got, let me know :-)
